src/ui/Editor.js

```diff
@@ -144,7 +145,7 @@ class Editor extends React.PureComponent<any, any, any> {
         editable: this._isEditable,
         nodeViews: boundNodeViews,
         state: editorState || EDITOR_EMPTY_STATE,
-        transformPastedHTML: normalizeHTML,
```

Comment:
The name `patchPastedHTML`  implies that user could monkey patch the HTML pasted, which seems hacky.

To make the editor truly customize-able, we could just let developer use custom `transformPastedHTML`  function (as describe at https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedHTML)

Note that the current implementation of `normalizeHTML` assumes the HTML is mostly generated from Google Doc, which is not always the case. 

Developer may want to integrate with her own normalization logic and choses not to use the default `normalizeHTML` function.

My recommendation is to let use opt out using `normalizeHTML` with  `transformPastedHTML`.

**For example:**

```
render(): void {
  return (
    <Editor
      transformPastedHTML={myNormalizeHTML}
    />
  );
}
```